src/frontend/src/pages/HomePage/WorkPackagesByTimelineStatus.tsx

```diff
@@ -61,24 +94,41 @@ const WorkPackagesByTimelineStatus: React.FC = () => {
             <Card
               variant="outlined"
               key={wbsPipe(wp.wbsNum)}
-              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default }}
+              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default, borderRadius: '16px' }}
             >
-              <CardContent sx={{ padding: 3 }}>
-                <Link
-                  variant="h6"
-                  component={RouterLink}
-                  to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}
-                  sx={{ marginBottom: 2 }}
-                >
-                  {wbsPipe(wp.wbsNum)} - {wp.name}
-                </Link>
-                <DetailDisplay label="End Date" content={datePipe(wp.endDate)} paddingRight={2} />
-                <DetailDisplay label="Progress" content={percentPipe(wp.progress)} paddingRight={2} />
-                <DetailDisplay label="Engineering Lead" content={fullNamePipe(wp.projectLead)} paddingRight={2} />
-                <DetailDisplay label="Project Manager" content={fullNamePipe(wp.projectManager)} paddingRight={2} />
-                <Typography>
+              <CardContent sx={{ padding: 2 }}>
+                <Grid container spacing={2}>
+                  <Grid item xs={9}>
+                    <Typography variant="h6" sx={{ textDecoration: 'underline', align: 'right' }}>
+                      {wp.wbsNum.projectNumber} - {wp.projectName}
+                    </Typography>
+                    <Link
+                      variant="h5"
+                      component={RouterLink}
+                      to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}
+                      sx={{ marginBottom: 2 }}
+                      noWrap={true}
+                    >
+                      {wbsPipe(wp.wbsNum)} - {wp.name}
+                    </Link>
+                    <Typography sx={{ marginTop: 1 }}>
+                      {datePipe(wp.startDate) + ' |-----' + wp.duration + 'wks------| ' + datePipe(wp.endDate)}
+                    </Typography>
+                    <Chip sx={{ marginTop: 1 }} icon={<Construction />} label={fullNamePipe(wp.projectLead)} />
+                    <Chip sx={{ marginTop: 2, marginLeft: 2 }} icon={<Work />} label={fullNamePipe(wp.projectManager)} />
+                  </Grid>
+
+                  <Grid item xs={3}>
```

Comment:
Ohhhhh you did it as two grid items like a left and a right, If your gonna do that then you need to make a container for the left and a container for the right and have them be items in another grid container, and then every row you want in each respective containers needs to be an item. Or you can just say that this progress bar is on the same level as the worpackage title and then the status is on the same level as the project title.